platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/FilterInjector.java

```diff
@@ -117,19 +116,14 @@ private void checkForMissedServletContexts() {
 
         if (service.getFilterRegistration(DELEGATING_FILTER) == null) {
           LOGGER.error(
-              "Platform filter delegate failed to start in time to inject itself into {} {}. This means the {} servlet will not properly attach the user subject to requests. Attempting to resolve the issue by restarting the bundle.",
+              "Platform filter delegate failed to start in time to inject itself into {} {}. This means the {} servlet will not properly attach the user subject to requests. A system restart is recommended.",
```

Comment:
✏️Given that this problem could cause all sorts of things not to work, I wonder if we shouldn't make these messages (here and in the catch block) a little stronger, maybe something like "A system restart is required". Also, could we report this to the admin using an alert?